news/2024/2024-01-15-osutaiko-osucatch-osumania-4k-world-cup-staff-applications.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: osu!taiko and osu!catch World Cup 2024 Staff Applications
+date: 2024-01-15 12:00:00 +0000
+---
+
+As the 2024 osu! World Cup kicks off with the osu!mania 7K World Cup, preparations for the next World Cups are already underway. Read on to find out how you can contribute!
+
+![](/wiki/shared/news/banners/OWC_Master_Logo.png)
+
+As with every year, we rely heavily on community members for help with many aspects of tournament production. This time, we're mainly looking for mappers and playtesters for the modes listed in the next section.
+
+Mappers will be tasked with creating beatmaps (many times of tailor made, yet-to-be-released [osu! originals](/wiki/osu!_originals)) based on a specification given by each mode's mappooling team. Mappers will be expected to incorporate feedback from the mappoolers and/or playtesters in an iterative process to refine the map until it is in a shippable state.
+
+Playtesters will be responsible for playing and giving feedback for a plethora of beatmaps (as requested by the mappooling team) to gauge aspects such as difficulty, suitability for a certain slot, skill representativity among others (specific workflows vary for each team). Playtesters are generally expected to have considerable skill on their respective game mode.
+
+Please note that if you are accepted into either team, you will not be eligible to participate as a player in that tournament.
```

Comment:
while what "that" refers to is understandable it's slightly incorrect to use this way

(as in an appropriate response to this sentence is "what tournament?")

```suggestion
Please note that if you are accepted into either team, you will not be eligible to participate as a player in the corresponding tournament.
```

Reply:
oh yeah definitely, I wasn't sure how to word this entire section